### ISSUE TITLE
Plot accuracy & loss graphs

### ISSUE THREAD

**Issue by toml01** [BioCellColman/BioCell#8]
Training + validation sets
- [ ] periodically
- [ ] end of training

_ref: https://keras.io/visualization/_
